tests/characters/control/blockchain/test_rpc_control_blockchain.py

```diff
@@ -56,6 +63,14 @@ def test_alice_rpc_character_control_create_policy(alice_rpc_test_client, create
         assert rpc_response.success is True
         assert rpc_response.id == response_id
 
+def test_alice_rpc_character_control_bad_input(alice_rpc_test_client, create_policy_control_request):
```

Comment:
do we only test bad input for alice? what about others?

Reply:
It's not really in this PR's scope for me to write tests for every single character's input.  I'm just testing the implementation.  The code paths are the same for all the other characters so this test covers all of them to some degree.

Reply:
It feels like our list of follow-ups to this PR is growing. If not done already, perhaps creating and linking follow-up issue(s) would be best.
1. CLI parameter thing we talked about during the walkthrough
2. Expanded character tests, or modifying the tests to be generic/repeatable for other character specifications.